lighthouse-cli/test/smokehouse/smokehouse.js

```diff
@@ -212,39 +139,49 @@ async function runSmokeTest(testOptions) {
       // Clear the network requests so that when we retry, we don't see duplicates.
       if (takeNetworkRequestUrls) takeNetworkRequestUrls();
 
-      logChildProcessError(localConsole, e);
+      logChildProcessError(bufferedConsole, e);
       continue; // Retry, if possible.
     }
 
     // Assert result.
-    report = getAssertionReport(result, expectation, {isDebug});
+    report = getAssertionReport(result, expectations, {isDebug});
     if (report.failed) {
-      localConsole.log(`${report.failed} assertion(s) failed.`);
+      bufferedConsole.log(`  ${report.failed} assertion(s) failed.`);
       continue; // Retry, if possible.
     }
 
     break; // Passing result, no need to retry.
   }
 
+  bufferedConsole.log(`  smoketest results:`);
+
   // Write result log if we have one.
-  if (result) {
-    localConsole.write(result.log);
-  }
+  if (result) bufferedConsole.write(result.log);
+
+  // If there's not an assertion report, not much detail to share but a failure.
+  if (report) bufferedConsole.write(report.log);
```

Comment:
```suggestion
```

Reply:
not sure why this suggestion? `report` and `result` too similar looking maybe?

(I was actually thinking of renaming them because they are a little hard to differentiate by name, but didn't want to add to the churn. If this is n=2 confusing, happy to rename now :)

Reply:
Wow I can't read haha. I thought this was a duplicate of L159.